packages/app/src/file-system/dto/get-paths.input.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { FileFilter, OpenDialogOptions } from "electron"
+
+export interface GetPathInput {
```

Comment:
Shouldn't it be called `GetPathsInput` (`Paths` vs `Path`) and the file be called something like `get-paths-input.object.ts` (just to follow the convention from other modules)?

Reply:
https://github.com/mudita/mudita-center/pull/907/commits/9c0bda2b2d75f8e396e1c22a11ac4efc41b4ef67